dist_ae1389-a

```diff
@@ -0,0 +1,18 @@
+class Dist_ae1389(BaseDistribution):
+def __init__(self):
+self.f_max = np.sin(np.pi*np.cos(x**2))
+self.x_min = 0
+self.x_max = 60
```

Comment:
Yes i'll resend it . I just copy pasted it to word because canopy crashed.
I didn't notice sorry about that

On Monday, April 11, 2016, Kyle Cranmer <notifications@github.com> wrote:

> In dist_ae1389-a
> <https://github.com/cranmer/intro-exp-phys-II/pull/51#discussion_r59199477>
> :
>
> > @@ -0,0 +1,18 @@
> > +class Dist_ae1389(BaseDistribution):
> > +def __init__(self):
> > +self.f_max = np.sin(np.pi*np.cos(x**2))
> > +self.x_min = 0
> > +self.x_max = 60
>
> this isn't valid python, you need to have indentions. Did you try to test
> it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cranmer/intro-exp-phys-II/pull/51/files/d902a075c59732b088ea07c9e373251419e80eac#r59199477>
>